### ISSUE TITLE
Accessibility: improving the HTML of IPython page according to WAVE tool

### ISSUE THREAD

**Issue by yamakat** [jupyterhub/outreachy#62]
### What page is this for?

https://ipython.org/usersurvey2011.html

### WAVE accessibility report

![screenshot indicating a summary from WAVE evaluation: there are 1 error, 30 contrast errors, 35 alerts, 2 features, 21 structural elements and 1 aria](https://images2.imgbox.com/a1/3b/0EqXwbyV_o.png)

### Errors

* 1x Language missing or invalid
* 30x Very low contrast

### Alerts

* 27x justified text
* 5x suspicious alternative text
* 2x image with title
* 1x suspicious link text

### Features

* 2x linked image with alternative text
* 21x structural elements
* 1x aria label 

### The HTML element to be changed

```html
1. <html> without attribute `lang`

2. increase `font-size` in CSS for better legibility

3. `color: #ce5c00` of links

4. `<a class="headerlink" href="#ipython-user-survey-2011" title="Permalink to this headline" style="color: rgb(221, 221, 221); background-color: rgb(255, 255, 255);">`

5. `text-align: justify` from the tag `<body>`

6. `<img class="logo" src="_static/IPy_header.png" alt="Logo">`

7. `<img alt="_images/countries.png" src="_images/countries.png">`

8. `<img alt="_images/platforms.png" src="_images/platforms.png">`

9. `<img alt="_images/partsused.png" src="_images/partsused.png">`

10. `<img alt="_images/sector.png" src="_images/sector.png">`

11. `<img alt="_static/nbviewer-thm-1.png" title="IPython Notebook Viewer">`
(also, the link is not working)

12. `<img id="sidebar-book-cover" title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." style="margin-bottom: 10px;" src="_static/learning-jupyter-book.png" width="160" height="200">`

13. `<a class="reference external" href="https://docs.google.com/spreadsheet/ccc?key=0AqIElKUDQl8tdDMzREtNSk9pX282N3lxaXhiTmxyN0E&authkey=CNOmu-QC&hl=en_GB&authkey=CNOmu-QC" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">here</a>`
```

### Your proposed HTML (or CSS), after change

```html
1. including HTML lang attribute: `<html lang="en">`

2. `font-size: 18`

3. `color: #8f3f00` of links, `a`s (or any other darker shade of orange, to increase contrast from background color)

4. class `headerlink` to `"color: rgb(51, 51, 51); background-color: rgb(242, 242, 242);">`

5. `text-align: left`

6. <img class="IPython_header" src="_static/IPy_header.png" alt="IPython logo: IP[y]: IPython Interative Computing">

7. `<img alt="Pie chart illustrating the number in percentages of respondents according to their countries, with USA having the majority of them, followed by the UK, Germany, France, Canada, Austria and Spain" src="_images/countries.png">`

8. `<img alt="Bar graph demonstrating users from different platforms of IPhyton: Windows, Mac OS X, Linux and Other" src="_images/platforms.png">`

9. `<img alt="Bar graph that depicts usage of IPython: Interactive shell, Parallel computing and Other, close to 30%" src="_images/partsused.png">`

10. `<img alt="Bar graph stating that the IPython is used mostly for academic reasons, and for, Industry and Hobby" src="_images/sector.png">`

11. `<img alt="Example of a graph used in Matplotlib" title="IPython Notebook Viewer" src="correctlink.com">`

12. `<img id="sidebar-book-cover-jupyter-for-all" alt="Cover of the book Jupyter Notebook for All - Part II: Interactive computing made easy" title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." style="margin-bottom: 10px;" src="_static/learning-jupyter-book.png" width="160" height="200">` (the ID would have to be changed as well, since the alt changes according to the book)

13. `<a class="reference external" href="https://docs.google.com/spreadsheet/ccc?key=0AqIElKUDQl8tdDMzREtNSk9pX282N3lxaXhiTmxyN0E&authkey=CNOmu-QC&hl=en_GB&authkey=CNOmu-QC" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">in this external spreadsheet</a>.`
```

### More info

I didn't create a comment regarding this evaluation in #38; [I did a comment on another page](https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1277794033), though (a simpler one).

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
